src/VirtualClient/VirtualClient.Contracts/Parser/MetricsParser.cs

```diff
@@ -20,5 +22,10 @@ public MetricsParser(string rawText)
             : base(rawText)
         {
         }
+
+        /// <summary>
+        /// Metrics list
+        /// </summary>
+        public List<Metric> Metrics { get; set; } = new List<Metric>();
```

Comment:
Yang, why do we need this? The MetricsParser already returns the set of metrics. This seems like overkill